src/app/map/map.component.ts

```diff
@@ -71,7 +71,7 @@ import { FilterResultsComponent } from '@app/filter-results/filter-results.compo
 export class MapComponent implements OnInit {
     @ViewChild(FilterResultsComponent)
     filterResultsComponent: FilterResultsComponent;
-    public removable = true;
+    // public removable = true;

```

Comment:
If this isn't being used lets go ahead an remove it.

If you'd like to leave it in for some future reason just add an explanation comment.